qemu/tests/cfg/scsi_mq_check.cfg

```diff
@@ -0,0 +1,14 @@
+- scsi_mq_check:
+    # As WinXP, Win2003, Win2008, Win7 not support multi queue. So don't test them.
+    no WinXP Win2003 Win2008 Win7
+    no RHEL.4 RHEL.5 RHEL.6
+    only virtio_scsi
+    type = scsi_mq_check
+    driver_name = vioscsi
+    # for linux:
+    pattern = "PCI-MSI-edge\s+virtio0-request"
+    check_queue_cmd = "cat /proc/interrupts"
+    # for windows:
+    Windows:
+        pattern = "instance.QueuesCount=(\d+)"
+        check_queue_cmd = "WIN_UTILS:\vioscsi.vbs & timeout 5 & type vioscsi.log"
```

Comment:
If it's better use && instead of & ,as we should have the right result and then run the next cmd.

Reply:
that's useful, will update, thanks